box.json

```diff
@@ -9,13 +8,12 @@
     ],
     "finder": [
         {
-            "name": "*.php",
+            "name": "{\\.(php|bash|fish|zsh)}",
```

Comment:
These are autocompletion dumps required to avoid a Symfony pseudo-bug: https://github.com/phpactor/phpactor/pull/2566